src/handlers/int-test-support/helpers/envHelper.ts

```diff
@@ -8,3 +8,12 @@ export const configStackName = (): string => `di-btm-cfg-${configName()}`;
 
 export const runViaLambda = (): boolean =>
   process.env.TEST_VIA_LAMBDA === "true";
+
+export const outputQueue = (): string => {
+  if (
+    process.env.OUTPUT_QUEUE_URL === undefined ||
+    process.env.OUTPUT_QUEUE_URL.length === 0
```

Comment:
And while we're at it, we should refactor our handlers to have an init-phase which is triggered on first function start-up as described [here](https://serverlessfirst.com/function-initialisation/). We could also load the required configs in this init-functions. @gdsgj basically did almost the same thing in his caching approach already and it seems to be a accepted pattern in Lambda to use global code to do some init-stuff. I'd prefer they'd explicitly call an init()-function on startup, but I can live with this approach. :-)

Reply:
Yeah I was thinking similar things, seems like every handler starts with a block of if this throw that, would be nice to tidy that up.

Reply:
I've taken this helper out and moved the check over to the handler. I'll have a think about an init function refactor